parlai/core/logging_utils.py

```diff
@@ -0,0 +1,129 @@
+#!/usr/bin/env python3
+
+# Copyright (c) Facebook, Inc. and its affiliates.
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+
+import os
+import sys
+import logging
+from pathlib import Path
+from datetime import date
+
+INFO_LEVEL = logging.INFO
```

Comment:
I suggest we drop the `_LEVEL` and keep it consistent with the logging constants. Then you can also change to `from logging import INFO, DEBUG, ...`